Get_Repo_Details/get_repo_details.py

```diff
@@ -0,0 +1,49 @@
+# Script to GET Repo details from user
+# @Author - Sandip Dutta
```

Comment:
We will have all of the contributors and maintainers on the documentation page. This is not required in the scripts as of now

Reply:
Yes I will remove these lines from the script.